source/fundamentals/auth.txt

```diff
@@ -231,33 +230,90 @@ connect using ``MONGODB-CR``.
 
 .. note::
 
-   The MONGODB-AWS authentication mechanism is available in MongoDB
-   Atlas.
+   The MONGODB-AWS authentication mechanism is available for MongoDB
+   deployments on MongoDB Atlas.
 
 The ``MONGODB-AWS`` authentication mechanism uses your Amazon Web Services
 Identity and Access Management (AWS IAM) credentials to authenticate your
-user.
+user. To learn more about configuring MongoDB Atlas, see the
+:atlas:`Set Up Passwordless Authentication with AWS IAM Roles </security/passwordless-authentication/#set-up-passwordless-authentication-with-aws-iam-roles>`
+guide.
 
-You can store your AWS credentials as environment variables, or insert
-them inline like the examples below. The driver checks for your credentials
-in the following order:
+You can authenticate using the ``MONGODB-AWS`` mechanism by providing
+your AWS IAM credentials in one of the following ways:
 
-1. Supplied values in a ``MongoCredential`` object or the provided connection string.
-2. Your environment variables. (``AWS_ACCESS_KEY_ID``, ``AWS_SECRET_ACCESS_KEY``,
-   and optionally ``AWS_SESSION_TOKEN``)
-3. The AWS EC2 endpoint specified in the ``AWS_CONTAINER_CREDENTIALS_RELATIVE_URI``
-   environment variable.
-4. The default AWS EC2 endpoint. For more information, see `IAM Roles for Tasks
-   <https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-iam-roles.html>`__
+1.  Use the AWS SDK (v1 or v2) to authenticate an IAM role.
+#. Specify your AWS credentials in a ``MongoCredential`` instance or connection
+   string.
+#. Specify your AWS credentials in environment variables.
+#. Specify an ECS/EC2 endpoint that hosts your container credentials.
 
+.. important::
 
-The following code snippets show how to specify the authentication mechanism,
-using the following placeholders:
+   The driver uses the preceding order to check for your credentials and uses
+   the first method that it finds.
+
+To learn how to refresh your AWS IAM credentials, see the
+:ref:`<java-mongodb-aws-refresh-credentials>` section.
+
+.. _java-mongodb-aws-sdk:
+
+Authenticate using the AWS SDK
++++++++++++++++++++++++++++++++
+
+You can authenticate with your AWS IAM role using the `AWS SDK for Java <https://docs.aws.amazon.com/sdk-for-java/index.html>`__
```

Comment:
Issue: Per [style guide](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#term-using), "by using" would be clearer than "using" in this case. Otherwise it could mean that the AWS IAM role is using the AWS SDK for Java.

Suggestion:

```suggestion
You can authenticate with your AWS IAM role by using the `AWS SDK for Java <https://docs.aws.amazon.com/sdk-for-java/index.html>`__
```